CIP-0116/cardano-babbage.json

```diff
@@ -0,0 +1,1952 @@
+          "value": {
+            "$ref": "cardano-babbage.json#/definitions/TransactionMetadatum"
+          }
+        },
+        "required": [
+          "key",
+          "value"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "AuxiliaryDataHash": {
+      "title": "AuxiliaryDataHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{64}$"
+    },
+    "AuxiliaryData": {
+      "title": "AuxiliaryData",
+      "type": "object",
+      "properties": {
+        "metadata": {
+          "$ref": "cardano-babbage.json#/definitions/TransactionMetadata"
+        },
+        "native_scripts": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/NativeScript"
+          }
+        },
+        "plutus_scripts": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/PlutusScript"
+          }
+        }
+      },
+      "required": [],
+      "additionalProperties": false
+    },
+    "TransactionBody": {
+      "title": "TransactionBody",
+      "type": "object",
+      "properties": {
+        "auxiliary_data_hash": {
+          "$ref": "cardano-babbage.json#/definitions/AuxiliaryDataHash"
+        },
+        "inputs": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/TransactionInput"
+          }
+        },
+        "outputs": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/TransactionOutput"
+          }
+        },
+        "fee": {
+          "$ref": "cardano-babbage.json#/definitions/UInt64"
+        },
+        "certs": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/Certificate"
+          }
+        },
+        "collateral": {
+          "title": "Collateral Inputs",
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/TransactionInput"
+          }
+        },
+        "collateral_return": {
+          "allOf": [
+            {
+              "$ref": "cardano-babbage.json#/definitions/TransactionOutput"
+            },
+            {
+              "title": "Collateral Return",
+              "description": "Collateral return, introduced in CIP-40"
+            }
+          ]
+        },
+        "mint": {
+          "$ref": "cardano-babbage.json#/definitions/Mint"
+        },
+        "network_id": {
+          "$ref": "cardano-babbage.json#/definitions/NetworkId"
+        },
+        "reference_inputs": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/TransactionInput"
+          }
+        },
+        "required_signers": {
+          "title": "Required signers",
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/Ed25519KeyHash"
```

Comment:
shouldnt this be credential rather than keyHash?

Reply:
It should be a keyhash. It is impossible to sign something with a script.